d2game/d2player/game_controls.go

```diff
@@ -621,6 +617,7 @@ func (g *GameControls) togglePartyPanel() {
 }
 
 func (g *GameControls) onCloseHeroStatsPanel() {
+	g.updateLayout()
```

Comment:
Good catch,
I had to miss it in my game controls refactor,
maybe, because I used to open panels with hotkeys :grinning: